custom/conf/app.example.ini

```
@@ -1384,6 +1384,12 @@ ROUTER = console
 ;; repo indexer by default disabled, since it uses a lot of disk space
 ;REPO_INDEXER_ENABLED = false
 ;;
+;; repo indexer skip forks from being indexed if setting is enabled
+;REPO_INDEXER_SKIP_FORKS = false
+;;
+;; repo indexer skip mirrors from being indexed if setting is enabled
+;REPO_INDEXER_SKIP_MIRRORS = false
```

Hmm…
I don't think this is the best approach.
What I imagine to be much more future proof would be something like
```ini
;; Define repo types that won't be indexed by the repo indexer
;; Possible values are fork, mirror, [in the future] username-matches:$REGEX, reponame-matches:$REGEX
;REPO_INDEXER_SKIPPED_REPOS = fork, mirror, [in the future] username-matches:.*test-user, reponame-matches:.*repo-?\d*
```
so, for the current state:
```suggestion
;; Define repo types that won't be indexed by the repo indexer
;; Possible values are fork and mirror
;REPO_INDEXER_SKIPPED_REPOS =
```

---

Mixing types and name matching looks like not a good idea. Maybe one option for types, one for name matching.